backend/src/main/java/com/bakdata/conquery/apiv1/QueryProcessor.java

```diff
@@ -525,7 +538,7 @@ public Stream<Map<String, String>> resolveEntities(Subject subject, List<FilterV
 			throw new ConqueryError.ExecutionProcessingTimeoutError();
 		}
 
-		if (execution.getState() == ExecutionState.FAILED) {
+		if (execution.getState(namespace.getExecutionManager()) == ExecutionState.FAILED) {
```

Comment:
wäre es nicht möglich den executionManager schon in der execution zu haben? Dann sparst du dir das wiederholte reingeben

Reply:
Ist jetzt injected